src/encoded/static/components/static-pages/HomePage.js

```diff
@@ -109,109 +98,419 @@ export default class HomePage extends React.PureComponent {
+                                        <div className="h-100 p-0 browse-data-collection-block browse-by-publication">
+                                            <a href="/search/?type=Publication&sort=static_content.location&sort=-number_of_experiment_sets&number_of_experiment_sets.from=1" className="h-100 d-flex flex-column text-decoration-none">
+                                                <div className="mt-2"><i className="icon icon-book-open fas"></i></div>
+                                                <div className="flex-grow-1 mt-2 browse-data-collection-block-title">Browse By Publication</div>
+                                                <div className="mt-2 pt-2 pb-2 pl-4 pr-4 browse-data-collection-block-desc">
+                                                    View Publications<br /> in the 4D Nucleome Database
+                                                </div>
+                                            </a>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                    <div className="col-12 col-md-5 pl-lg-5 d-flex flex-column browse-data-collection-col-collections">
+                        <div className="flex-grow-1 mt-1">
+                            <h2 className="homepage-section-title new-design text-500">4DN Data Collections</h2>
+                            <a href="/hic-data-overview" className="btn btn-primary w-100">
+                                Hi-C Datasets
+                            </a>
+                            <a href="/microscopy-data-overview" className="btn btn-primary w-100 mt-05">
+                                All Microscopy Datasets
+                            </a>
+                            <a href="/resources/data-collections/chromatin-tracing-datasets" className="btn btn-primary w-100 mt-05">
+                                Chromatin Tracing Datasets
+                            </a>
+                            <a href="/resources/data-collections" className="btn btn-primary w-100 mt-1 btn-all-data-collections">
+                                <span className="float-left ml-1">View All Data Collections</span>
+                                <span className="float-right mr-1"><i className="icon icon-arrow-right fas"></i></span>
+                            </a>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    );
+});
 
-    constructor(props){
+class RecentlyReleasedDataSets extends React.PureComponent {
+    constructor(props) {
         super(props);
-        this.handleMouseEnter = this.handleMouseEnter.bind(this);
-        this.handleMouseLeave = this.handleMouseLeave.bind(this);
+        this.state = {
+            'mounted': false,
+        };
```

Comment:
It would be nice to add the pieces of state used on line 315 (and any other ones used in this component) to the constructor with a default value.